src/packages/utils/ratlib.py

```diff
@@ -46,6 +46,17 @@ def colorize(self) -> str:
             text_color = Colors.PURPLE
         return bold(color(self.value, text_color))
 
+    def plaintext(self) -> str:
```

Comment:
This function seems entirely redundant?

This function doesn't do anything with `text_color`, and returns something that can be accessed directly.